package.json

```diff
@@ -67,11 +67,6 @@
         "git add"
       ],
       "*.ts": [
-        "tslint --fix",
-        "prettier --write",
-        "git add"
-      ],
-      "*.{yml,json,md,html}": [
```

Comment:
why did you remove yml, json, md, and html from being formatted by prettier?

Reply:
The formatter doesn't work on Windows and messes up the formatting every time. I can add it back in if you like; just I have to comment it out when I'm committing as I can't use it.

Reply:
that's weird - but if you are saying so, I'm okay with removing it